public/pages/Dashboard/Components/AnomaliesDistribution.tsx

```diff
@@ -156,85 +162,82 @@ export const AnomaliesDistributionChart = (
             title={finalDetectors.length}
             isLoading={anomalyResultsLoading}
             titleSize="s"
+            style={{ color: '#000' }}
```

Comment:
Why not use default color?

Reply:
I recalled this is required by UX to change the color, and I think that's why I use 000 for content-panel-style. I will double confirm wit UX on this.

Reply:
Have confirmed with UX on this. We are good to use default color.